src/main/scala/friday/tetraNumber.scala

```diff
@@ -0,0 +1,14 @@
+package friday
+import scala.util.control.Breaks._
+
+object tetraNumber extends App {
```

Comment:
I think you missed the whole point of exercise. You have to calculate the base of the pyramid for the given tetrahedral number. Where you're going from 0 to the upper bound of it printing them...